src/code/z_actor.c

```diff
@@ -3856,19 +3856,19 @@ s16 func_80034DD4(Actor* actor, GlobalContext* globalCtx, s16 arg2, f32 arg3) {
     return arg2;
 }
 
-void func_80034EC0(SkelAnime* skelAnime, struct_80034EC0_Entry* animations, s32 index) {
+void Animation_ChangeInfo(SkelAnime* skelAnime, AnimationInfo* animationInfo, s32 index) {
```

Comment:
this name implies to me that its changing something about the info and not the animation itself. I dont have any better suggestions right now but will think about it more

Reply:
What about `Animation_ChangeByInfo`?

Reply:
`Using` or `From`'d probabably be better

Reply:
I went with `Animation_ChangeByInfo` for its shorter length